lib/hiera.rb

```diff
@@ -5,6 +5,7 @@ class Hiera
   require "hiera/version"
   require "hiera/config"
   require "hiera/util"
+  require "hiera/util/win32" if !!File::ALT_SEPARATOR
```

Comment:
Since win32 is guarded by [`File::ALT_SEPARATOR`](https://github.com/puppetlabs/hiera/pull/393/files#diff-d039e02fd0e21a94fc3125dd21ba6b89R4), can we just require `hiera/util/win32` unconditionally here and so we don't have to [`require_relative`](https://github.com/puppetlabs/hiera/pull/393/files#diff-cfd287e5017035e0b2e21c032fd4409fR151) above.